### ISSUE TITLE
error when using expired cached session

### ISSUE THREAD

**Issue by machnee** [Geotab/mygeotab-python#92]
I'm occasionally getting an error using the myg console, and I'm suspecting it's due to an expired session.  I'm launching the console with the full arguments including username and password.
My workaround at the moment is to delete the entry from the config.ini

Thanks for putting this module here, it's been incredibly useful so far.

```
myg console <database_name> --user <user> --password <password>

Traceback (most recent call last):
  File "/usr/local/lib/python3.4/site-packages/mygeotab/api.py", line 94, in call
    result = _query(self._server, method, params, self.timeout, verify_ssl=self._is_verify_ssl)
  File "/usr/local/lib/python3.4/site-packages/mygeotab/api.py", line 308, in _query
    return _process(response.json(object_hook=object_deserializer))
  File "/usr/local/lib/python3.4/site-packages/mygeotab/api.py", line 320, in _process
    raise MyGeotabException(data['error'])
mygeotab.exceptions.MyGeotabException: InvalidUserException
Incorrect login credentials @ \u0027<database_name>\u0027

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/bin/myg", line 11, in <module>
    sys.exit(main())
  File "/usr/local/lib/python3.4/site-packages/click/core.py", line 722, in __call__
    return self.main(*args, **kwargs)
  File "/usr/local/lib/python3.4/site-packages/click/core.py", line 697, in main
    rv = self.invoke(ctx)
  File "/usr/local/lib/python3.4/site-packages/click/core.py", line 1066, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/local/lib/python3.4/site-packages/click/core.py", line 895, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/local/lib/python3.4/site-packages/click/core.py", line 535, in invoke
    return callback(*args, **kwargs)
  File "/usr/local/lib/python3.4/site-packages/click/decorators.py", line 27, in new_func
    return f(get_current_context().obj, *args, **kwargs)
  File "/usr/local/lib/python3.4/site-packages/mygeotab/cli.py", line 203, in console
    local_vars = _populate_locals(database, password, server, session, user)
  File "/usr/local/lib/python3.4/site-packages/mygeotab/cli.py", line 242, in _populate_locals
    api.get('User', name=session.credentials.username)
  File "/usr/local/lib/python3.4/site-packages/mygeotab/api.py", line 137, in get
    return self.call('Get', type_name=type_name, **parameters)
  File "/usr/local/lib/python3.4/site-packages/mygeotab/api.py", line 101, in call
    self.authenticate()
  File "/usr/local/lib/python3.4/site-packages/mygeotab/api.py", line 192, in authenticate
    verify_ssl=self._is_verify_ssl)
  File "/usr/local/lib/python3.4/site-packages/mygeotab/api.py", line 308, in _query
    return _process(response.json(object_hook=object_deserializer))
  File "/usr/local/lib/python3.4/site-packages/mygeotab/api.py", line 320, in _process
    raise MyGeotabException(data['error'])
mygeotab.exceptions.MyGeotabException: ArgumentException
Must provide either password or session ID
```

**Comment by aaront** [comment]
@machnee I think this should be fixed in v0.8.0, which was just uploaded to PyPI

**Comment by machnee** [comment]
Works!  Thanks Aaron.

```
myg console <database> --user <user> --password <pass>
Your session has expired. Please login again.
Logging in...  [####################################]  100%
Logged in as: <user> @ myXX.geotab.com/<database>
MyGeotab Console 0.8.0 [Python 3.X.X]
Logged in as: <user> @ myXX.geotab.com/<database>
```
